src/eduid_webapp/email/helpers.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+from functools import wraps
+
+from flask import current_app, session
+
+from eduid_userdb import User
+from eduid_userdb.proofing import ProofingUser
+from eduid_userdb.exceptions import UserDoesNotExist, MultipleUsersReturned
+from eduid_common.api.utils import retrieve_modified_ts, get_user
+from eduid_common.api.exceptions import ApiException
+
+
+def require_user(f):
```

Comment:
It is fetched from the central db `get_user` only uses the local db to update the modified_ts attr.

I don't use the one from eduid_common.api.decorators b/c that one is not using retrieve_modified_ts. But I was saying in the mail, retrieve_modified_ts makes little sense now...